spec.bs

```diff
@@ -620,6 +624,10 @@ To <dfn>generate and score bids</dfn> given an [=auction config=] |auctionConfig
 1. Let |allBuyersExperimentGroupId| be |auctionConfig|'s
   [=auction config/all buyer experiment group id=].
 1. Let |auctionSignals| be |auctionConfig|'s [=auction config/auction signals=].
+1. If |auctionSignals| is a [=promise=]:
+  1. Let |auctionSignals| be the result of [=awaiting=] for |auctionSignals|.

```

Comment:
Please don't use "awaiting" but instead use "upon fulfillment" (and/or rejection if you need it too): https://dontcallmedom.github.io/webdex/u.html#upon%20fulfillment%40%40webidl%25%25dfn.

See this example: https://wicg.github.io/webusb/#ref-for-upon-fulfillment